src/core/runtime.d

```diff
@@ -313,13 +313,54 @@ struct Runtime
     }
 
 
+    /**
+     * Set source file path for coverage reports.
+     *
+     * Params:
+     *  path = The new path name.
+     */
+    static void coverSourcePath(string path)
```

Comment:
???
Nobody knows that coverage generation can be configured.
If there are function that allow to change the behavior, we need to make them accessible and document them.